client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/PlacesAdditionalForms/SchoolEventForm.tsx

```diff
@@ -70,25 +80,40 @@ const SchoolEventForm : React.FC = () : JSX.Element => {
                 </Grid>
                 {
                     grades.length > 0 &&
-                    <Grid item xs={2}>
+                    <>
+                    <Grid item xs={1}/>
```

Comment:
why do you need this here? I saw grid item used as sort of 'divider' or spacing the other items around the code, I think if you're using it as I said, I think you should just use a bigger xs or add some css